### ISSUE TITLE
Accessibility welcome menu, two missing translation strings

### ISSUE THREAD

**Issue by aha999** [UriahShaulMandel/BaldPhone#142]
**Describe the bug**

- In the Accessibility welcome menu (after installing the latest commits apk), i see i haven't translated words Long and Short, checked the provided strings and it seems the strings "Long" and "Short" are missing for translation.
